lib/util/git/types.ts

```diff
@@ -19,6 +19,7 @@ export interface StorageConfig {
   currentBranch?: string;
   defaultBranch?: string;
   url: string;
+  extraGitOptions?: GitOptions;
   extraCloneOpts?: GitOptions;
```

Comment:
Is this still used?

Reply:
yes, this is what the relevant change of this pr is, see lib/util/git/index.ts line 199 and line 444